instrumentation/failsafe-3.0/library/src/main/java/io/opentelemetry/instrumentation/failsafe/v3_0/FailsafeTelemetry.java

```diff
@@ -100,6 +100,7 @@ public <R> RetryPolicy<R> createRetryPolicy(RetryPolicy<R> delegate, String retr
         meter
             .histogramBuilder("failsafe.retry_policy.attempts")
             .setDescription("Number of attempts for each execution.")
+            .setUnit("{retry_attempt}")
```

Comment:
I don't think we have an established convention, but I think I'd go with
```suggestion
            .setUnit("{attempt}")
```